client/locales/en.json

```diff
@@ -1080,18 +1080,18 @@
     "frequentlyAskedQuestions": {
       "title": "Frequently Asked Questions",
       "labels": [
-        "How can I download the emoji?",
-        "How can I add the emoji to my Discord server?",
+        "How can I download an emoji?",
+        "How can I add an emoji to my Discord server?",
         "This emoji is inappropriate. What can I do?"
       ],
       "answers": [
         "You can use the button below to download <span> emoji.",
         {
-          "title": "I'm glad to hear that! I will tell you 3 methods for this.",
+          "title": "I'm glad to hear that! There are 3 ways to download an emoji.",
```

Comment:
"I'm glad to hear that!" cümlesini yukarıdaki bir değişiklikte silmişsin ama burada silmemişsin, ya sil ya da hiç silme

Reply:
hallettim